zetaclient/config/config.go

```diff
@@ -83,3 +93,12 @@ func GetPath(inputPath string) string {
 	}
 	return filepath.Join(path...)
 }
+
+func AnyBannedAddress(addrs ...string) bool {
```

Comment:
might be a good idea to a add a small unit test for this function

Reply:
Yeah. It's tested here: https://github.com/zeta-chain/node/blob/feat-zetaclient-banned-addresses/zetaclient/common/utils_test.go#L14